src/favouritesPage/favouritesPage.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import { Favourite } from '../favourites/types';
+import { RootState } from '../root/rootReducer';
+import { connect } from 'react-redux';
+import { Col, Container, Row } from 'react-grid-system';
+import { removeFavouriteTarget } from '../favourites/actions';
+import { useTranslation } from 'react-i18next';
+import FavouriteCard from './components/favouriteCard';
+import { PlotSearch } from '../plotSearch/types';
+import { fetchPlotSearches } from '../plotSearch/actions';
+import { LoadingSpinner } from 'hds-react';
+
+interface State {
+  favourites: Favourite;
+  plotSearches: PlotSearch[];
+  isFetchingPlotSearches: boolean;
+}
+
+interface Props {
+  plotSearches: PlotSearch[];
+  favourites: Favourite;
+  removeFavouriteTarget: (id: number) => void;
+  fetchPlotSearches: () => void;
+  isFetchingPlotSearches: boolean;
+}
+
+const FavouritesPage = (props: Props): JSX.Element => {
+  const { t } = useTranslation();
+  const handleTargetRemove = (id: number): void => {
+    props.removeFavouriteTarget(id);
+  };
+
+  const [plotSearch, setPlotSearch] = useState<PlotSearch | null>(null);
+
+  useEffect(() => {
+    props.fetchPlotSearches();
+  }, []);
+
+  useEffect(() => {
+    setPlotSearch(
+      props.plotSearches.filter(
+        (plotSearch) => plotSearch.id === props.favourites.plotSearch
+      )[0]
+    );
+  });
```

Comment:
The list of variables to trigger the effect on is missing, probably should be `[ props.favourites ]`.

Reply:
Hmm?

Reply:
aahh! true.

Reply:
Actually, `props.plotSearches` needs to also be added here; if the first view after a refresh to be loaded is the favourites view, this only triggers when no plot searches have been loaded yet.